application/core/LSYii_Validators.php

```diff
@@ -126,6 +126,27 @@ public function xssFilter($value)
                 'news' => true,
                 )
         );
+
+	$config = $filter->getPurifier()->config;
+        $config->set('HTML.DefinitionID', 'html5-definitions');
+        $def = $config->maybeGetRawHTMLDefinition();
+        $max = $config->get('HTML.MaxImgLength');
+        if ($def) {
+            $def->addElement('video', 'Inline', 'Flow', 'Common', array(
+                'controls' => 'Bool',
```

Comment:
Does this probably open up vulnerabilities like https://html5sec.org/#55 and https://html5sec.org/#56 ?

Reply:
Tested those and other potential vulnerabilities. They get filtered.